### DISCUSSION TITLE
Target API

### DISCUSSION THREAD

**Discussion by OrelsProjects** [DuCanhGH/next-pwa#159] [General]
Hello there :)

Google has asked me to target API 34, because my current PWA targets API 33.
How can I configure it?

Thanks!

**Comment by DuCanhGH** [comment]
@OrelsProjects this is not in the scope of `next-pwa`, and I haven't ever played around with publishing a PWA to Google Play Store, so I can't help you with much, unfortunately. But isn't it that you simply need to change `targetSdkVersion` to 34?
